app/views/users/show.index.erb

```diff
@@ -0,0 +1 @@
+<h1>Here is a list of Users for a given id</h1>
```

Comment:
- There is an error viewing this file, this is majorly because you have named the file incorrectly. The filename should be `show.html.erb`. Please fix this 🙏